src/BenchmarksDriver/Program.cs

```diff
@@ -444,6 +454,7 @@ public static int Main(string[] args)
                         [ConnectionFilter] [nvarchar](max) NULL,
                         [WebHost] [nvarchar](max) NOT NULL,
                         [KestrelThreadCount] [int] NULL,
+                        [KestrelThreadPoolDispatching] [bit] NULL,
```

Comment:
Both `KestrelThreadCount` and `KestrelThreadPoolDispatching` are nullable by design, since `null`, `true`, and `false` all have different meaning.  So the DB column should be nullable as well.